README.md

```diff
@@ -27,15 +27,17 @@ module.exports = {
       loaders: ['style-loader', 'css-loader'],
     }, {
       test: /\.css\.js$/,
-      loader: 'css-js-loader',
+      loaders: ['css-js-loader', 'value-loader'],
     }],
   },
 };
 ```
 
+NOTE: You don't need to chain with [value-loader] per-se, but doing so gives you caching, nested dependency monitoring/reloading and the ability to use webpack's module rollup.
```

Comment:
I haven't heard of webpack using "rollup" to describe this this feature. Stick with "tree shaking" or "unused module export detection" instead.